akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaRange.scala

```diff
@@ -67,13 +67,13 @@ object MediaRange {
       if (params.nonEmpty) params foreach { case (k, v) ⇒ r ~~ ';' ~~ ' ' ~~ k ~~ '=' ~~# v }
       r
     }
-    override def isApplication = mainType == "application"
-    override def isAudio = mainType == "audio"
-    override def isImage = mainType == "image"
-    override def isMessage = mainType == "message"
-    override def isMultipart = mainType == "multipart"
-    override def isText = mainType == "text"
-    override def isVideo = mainType == "video"
+    override def isApplication = mainType equalsIgnoreCase "application"
```

Comment:
Isn't there a more precise location where we can fix this, so we don't have to remember to sprinkle `equalsIgnoreCase` in several places?